scripts/jenkins/build

```diff
@@ -52,6 +55,35 @@ fi
 
 ./autogen.sh |& tee $ARTIFACTS/configure.log
 
+if [[ $TRANSPORT == "lnet" ]]
+then
+	TRANSPORT='lustre-client'
+else
+	echo "No need chage value"
```

Comment:
s/No need chage value/No need change value

But still, this sentence is confusing.
`No changes are needed`?